rts/Sim/Projectiles/Projectile.h

```diff
@@ -92,7 +92,7 @@ class CProjectile: public CExpGenSpawnable
 	// UNSYNCED ONLY
 	CMatrix44f GetTransformMatrix(bool offsetPos) const;
 
-	float GetSortDist() const { return sortDist; }
+	const float& GetSortDist() const { return sortDist; }
```

Comment:
can't use lvalue for std::tie

Reply:
perhaps that's just grounds for avoiding `std::tie` and using regular `std::make_tuple`? it sounds weird for this kind of getter to return a reference